packages/components/psammead-image-placeholder/package.json

```diff
@@ -17,11 +17,11 @@
   },
   "homepage": "https://github.com/bbc/psammead/blob/latest/packages/components/psammead-image-placeholder/README.md",
   "dependencies": {
-    "@bbc/psammead-assets": "^0.1.2",
-    "@bbc/psammead-styles": "^1.1.2"
+    "@bbc/psammead-assets": "^1.0.6",
```

Comment:
Has major bump, any expected changes?

Reply:
This is where test should focus their eyes

Reply:
# Psammead Assets Changelog

<!-- prettier-ignore -->
| Version | Description |
|---------|-------------|
| 1.0.6   | [PR#1179](https://github.com/bbc/psammead/pull/1179) use `psammead-test-helpers@1.0.2`|
| 1.0.5   | [PR#1082](https://github.com/bbc/psammead/pull/1082) Bump lodash security vulnerability |
| 1.0.4   | [PR#783](https://github.com/bbc/psammead/pull/783) Update to latest psammead-test-helpers. Update snapshots. |
| 1.0.3   | [PR#787](https://github.com/bbc/psammead/pull/787) Fix brand stories in Firefox high contrast mode |
| 1.0.2   | [PR#713](https://github.com/bbc/psammead/pull/713) Update `styled-components` to 4.3.2 |
| 1.0.1   | [PR#704](https://github.com/bbc/psammead/pull/704) Remove `fill` from SVGs |
| 1.0.0   | [PR#679](https://github.com/bbc/psammead/pull/679) Bump version number |
| 0.1.9   | [PR#490](https://github.com/bbc/psammead/pull/490) Update SVGs to consistent export, minify and add persian |
| 0.1.8   | [PR#444](https://github.com/bbc/psammead/pull/444) Add service SVG's |
| 0.1.7   | [PR#424](https://github.com/bbc/psammead/pull/424) Add Snyk badge to readme |
| 0.1.6   | [PR#323](https://github.com/BBC/psammead/pull/323) Update readme storybook badge |
| 0.1.5   | [PR#264](https://github.com/BBC/psammead/pull/319) Resolving package-lock issues. |
| 0.1.4   | [PR#224](https://github.com/BBC-News/psammead/pull/224) Add tests for the exported values, coverage 100% :tada: |
| 0.1.3   | [PR#212](https://github.com/BBC-News/psammead/pull/212) Update package description and README. |
| 0.1.2   | [PR#173](https://github.com/BBC-News/psammead/pull/173) Update PRs welcome link |

Reply:
Couldn't much changes in the `psammead-assets` changelog affecting Image placeholders as most of the changes are made to brand svg.